Dockerfile

```diff
@@ -0,0 +1,46 @@
+# для deploy
+FROM python:3.8
+
+WORKDIR /divorce_project
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update \
+    && apt-get install netcat -y \
+    && apt-get install -y dos2unix
+RUN apt-get upgrade -y && apt-get install postgresql gcc python3-dev musl-dev -y
```

Comment:
Each instruction will create a layer in the Docker image and will waste space, making your image big.
It's better to do in this way:
```bash
RUN apt-get update && apt-get upgrade -y  && apt-get install -y \
    netcat \
    dos2unix \
    postgresql \
    gcc \
    python3-dev \
    musl-dev
&& rm -rf /var/lib/apt/lists/*
```
Notice the last line - it will delete the temporary install packages and as it is in the same command as previous, there'll be no saved temporary data in Docker layer